src/sage/misc/decorators.py

```diff
@@ -25,15 +25,12 @@
 #                  http://www.gnu.org/licenses/
 # *****************************************************************************
 
-from functools import (partial, update_wrapper, WRAPPER_ASSIGNMENTS,
```

Comment:
Just formatting changes in this file. At some point while testing things I thought I was going to add type annotations to this file but I didn't end up needing to.